plc4j/protocols/s7/src/main/java/org/apache/plc4x/java/s7/netty/model/params/VarParameter.java

```diff
@@ -24,6 +24,13 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 import java.util.List;
 
+/**
+ * "Command" Message to inform PLC of wanted operation.
+ * Also contains {@link VarParameter#items} that hold detailed information on the "Command", e.g.,
+ * addresses to read or to write to.
+ *
+ * TODO 01.08.18 jf: Could this be renamed to Something like Command as this seems to be the command message pattern?
```

Comment:
These classes are driver-internal classes. No user should ever deal with them. The naming is correlated to names used in some of the documentation of the S7 protocol. So even if I agree that the correct name for this class from a pattern point of view might be different, I would suggest to keep it the way it is, because this way there is the correlation to the informal specs.

Reply:
Okay, the correlation makes sense.
But its good to know that my assumption was right.
It simply helped my understanding of the process to see them as that.